calculator.go

```diff
@@ -1,13 +1,94 @@
-// Package calculator provides a library for simple calculations in Go.
 package calculator
 
-// Add takes two numbers and returns the result of adding them together.
-func Add(a, b float64) float64 {
-return a + b
+import (
+	"errors"
+	"math"
+	"strconv"
+	"strings"
+)
+
+// Add takes a variable number of float64 and returns their sum.
+func Add(nums ...float64) float64 {
+	sum := 0.0
+	for _, num := range nums {
+		sum += num
+	}
+
+	return sum
+}
+
+// Subtract takes a variable number of float64 and returns their difference.
+func Subtract(nums ...float64) float64 {
+	difference := nums[0]
```

Comment:
This is an even worse problem, because if `nums` is empty or nil, then this panics!